### ISSUE TITLE
Rename cta -> ctao in observatory_locations?

### ISSUE THREAD

**Issue by morcuended** [gammapy/gammapy#5501]
https://github.com/gammapy/gammapy/blob/2a4d540738e6a25450c0655ca5d9b2b57903e91e/gammapy/utils/observers.py#L55

For both `cta_south` and `cta_north` and all references to these key names.

**Comment by bkhelifi** [comment]
Indeed, one needs to update the names and put a log.ingo message. Could you do it?

**Comment by Astro-Kirsty** [comment]
Are you still working on this @morcuended?

**Comment by morcuended** [comment]
No, I did not follow it up.
